common/djangoapps/microsite_configuration/microsite.py

```diff
@@ -66,7 +66,15 @@ def get_value_for_org(org, val_name, default=None):
     This returns a configuration value for a microsite which has an org_filter that matches
     what is passed in
     """
-    return BACKEND.get_value_for_org(org, val_name, default)
```

Comment:
Yes, I agree. When I add those ORG->microsite mapping tables, I'll look into moving this back into the backends